src/modules/wallet/modules/assets/less/assets.less

```diff
@@ -134,7 +136,9 @@
       margin: auto 30px auto auto;
       font-size: 17px;
       color: @color-submit-400;
-      line-height: 56px;
+      line-height: 57.5px;
```

Comment:
It would be better to put repeating values into local variables in the beginning of the file, for example:
```
// imports

@chart-line-height: 57.5px;

// code

selector {
  selector {
    line-height: @chart-line-height;
  }
}
```